### PR TITLE
Minor fixes for image builder script

### PR DESCRIPTION
**Pull request by iartemenko**
Make "parted" never prompts for user intervention.
Unmount partitions.

Signed-off-by: Iurii Artemenko <iurii_artemenko@epam.com>

### CODE REVIEW

**Comment by otyshchenko1** [comment]
tested-by otyshchenko1

**Comment by otyshchenko1** [comment]
Pls, don't merge right now. Although it works, a review feedback from other guys is needed.

File: doc/mk_sdcard_image.sh [hunk 1]
```diff
@@ -266,6 +266,7 @@ make_image()
 	print_step "Preparing image at ${img_output_file}"
 
 	sudo umount -f ${MOUNT_POINT} || true
+        ls ${img_output_file}?* | xargs -n1 sudo umount -l -f || true
```
Comment (lorc) [inline doc/mk_sdcard_image.sh]: Something is wrong with formatting there.

Comment (iartemenko) [reply doc/mk_sdcard_image.sh]: Done

**Pull request merged**